src/javascript/_common/base/client_base.js

```diff
@@ -262,13 +262,15 @@ const ClientBase = (() => {
 
     const getBasicUpgradeInfo = () => {
         const upgradeable_landing_companies = State.getResponse('authorize.upgradeable_landing_companies');
+        const landing_company_obj = State.getResponse('landing_company');
 
         let can_open_multi = false;
         let can_upgrade_to = [];
         let type;
         if ((upgradeable_landing_companies || []).length) {
             const current_landing_company = get('landing_company_shortcode');
-            can_open_multi = upgradeable_landing_companies.indexOf(current_landing_company) !== -1;
+            can_open_multi = upgradeable_landing_companies.indexOf(current_landing_company) !== -1 &&
+                             (landing_company_obj && landing_company_obj.legal_allowed_currencies);
```

Comment:
can you add a comment here about why this check works?

Reply:
Sure